modules/components/BlockView.tsx

```diff
@@ -3,21 +3,17 @@ import { useRef } from "react";
 import { ShaderMaterial } from "three";
 
 import vert from "@/modules/patterns/shaders/default.vert";
-import {
-  Pattern,
-  PatternParameter,
-  PatternParameters,
-} from "@/modules/common/types/Pattern";
 import { Block } from "../common/types/Block";
+import { PatternParams } from "../common/types/PatternParams";
 
-type BlockViewComponent<T extends PatternParameters> = React.FC<{
+type BlockViewComponent<T extends PatternParams> = React.FC<{
   block: Block<T>;
 }>;
 
-const BlockView: BlockViewComponent<PatternParameters> = ({ block }) => {
+const BlockView: BlockViewComponent<PatternParams> = ({ block }) => {
   const shaderMaterial = useRef<ShaderMaterial>(null);
 
-  useFrame(({ clock }) => {
+  useFrame(({ clock }, delta) => {
```

Comment:
`delta` doesn't seem to be used

Reply:
Aww yiss @milotoor in da house. We've been moving pretty fast, so you can expect more of this :D